core/templates/dev/head/pages/get_started/get_started.html

```diff
@@ -1,102 +1,118 @@
-    <li>
-      <span class="oppia-navbar-breadcrumb-separator"></span>
-      Get Started
-    </li>
-  </ul>
-{% endblock navbar_breadcrumb %}
-
-{% block content %}
-  <div class="oppia-static-header">
-    <h1 translate="I18N_GET_STARTED_PAGE_HEADING"></h1>
-  </div>
-
-  <background-banner></background-banner>
-  <div class="oppia-page-card oppia-static-content oppia-static-content-below-banner">
-    <div class="oppia-static-card-content oppia-static-card-content-narrow oppia-get-started">
-
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_1">
-      </p>
-
-      <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_2_HEADING"></h2>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_2">
-      </p>
-
-
-      <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_3_HEADING"></h2>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_3">
-      </p>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_4">
-
-      </p>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_5">
-
-      </p>
-      <ul style="margin-left: 40px;">
-        <li><a href="https://www.oppia.org/explore/0" target="new">Welcome to Oppia</a></li>
-        <li><a href="https://www.oppia.org/collection/wqCTKpKA0LBe" target="new">Get Electrified!</a></li>
-        <li><a href="https://www.oppia.org/collection/inDXV0w8-p1C" target="new">Programming with Carla</a></li>
-      </ul>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_6">
-      </p>
-
-      <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_7_HEADING"></h2>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_7">
-      </p>
-
-      <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_8_HEADING"></h2>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_8">
-      </p>
-
-      <!-- todo: make it easy to get the right link to share, update this doc with that info -->
-
-      <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_9_HEADING"></h2>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_9">
-      </p>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_10">
-
-      </p>
-
-      <h2 translate="I18N_GET_STARTED_PAGE_PARAGRAPH_11_HEADING"></h2>
-      <p translate="I18N_GET_STARTED_PAGE_PARAGRAPH_11">
-      </p>
-    </div>
-  </div>
-  <div class="oppia-footer-padding-below-banner">
-  </div>
-  <style>
-    .oppia-get-started h2 {
-      font-size: 1.17em;
-      font-weight: bold;
-      text-align: left;
-    }
-  </style>
-{% endblock %}
-
-{% block footer_js %}
-  {{ super() }}
-  <!-- This code is used for inserting webpack bundles
-     https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates -->
-  <% for (var chunk in htmlWebpackPlugin.files.js) { %>
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    <meta charset="UTF-8">
+
+    <!-- This code is used for inserting meta tags via webpack -->
+    <%= htmlWebpackPlugin.options.meta.toHTML() %>
```

Comment:
Very nice that you simplified this/moved to utils!

Reply:
Yeah, I noticed it was a pain to try to figure out what was happening there.